src/__tests__/components/__snapshots__/App.test.js.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`App Component matches the snapshot 1`] = `ShallowWrapper {}`;
```

Comment:
I think the __snapshots__ folder autogenerates every time we run tests. Shouldn't be placed in the .gitignore file.

Reply:
> I think the **snapshots** folder autogenerates every time we run tests. Shouldn't be placed in the .gitignore file.

No please @reifred, i have not placed it in the .gitignore file, clearly check please. Thank you

Reply:
> > I think the **snapshots** folder autogenerates every time we run tests. Shouldn't be placed in the .gitignore file.
> 
> No please @reifred, i have not placed it in the .gitignore file, clearly check please. Thank you

@Emmanuel-Dominic am not saying you placed in the .gitignore file rather am inquiring.

Reply:
> > I think the **snapshots** folder autogenerates every time we run tests. Shouldn't be placed in the .gitignore file.
> 
> No please @reifred, it shouldn't be place in the .gitignore file because TravisCI runs with the snapshots found rather it doesn't auto-generate them during testing and for that case before travis failed because it could not allocate the the snapshoot.

Thanks for that clarification.